### PR TITLE
Star Rating Component  #36 #19 #17 #10

### PR DESCRIPTION
**Pull request by andrewMacmurray**
- 5 stars render on current amazon view
- Have added [classnames module](https://github.com/JedWatson/classnames) to dependencies, this lets you add multiple classes to the 'className' attribute and has a great way to toggle the 'on' 'off' states of classes using booleans 
- Also have added js version of sampledbdata to frontend folder, webpack couldnt reach it and was also complaining about the json format (may need to add a loader to webpack to handle this later on)

### CODE REVIEW

File: frontend/src/js/Ratings/FiveStars.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    const reviews = JSON.parse(this.props.reviewsArray)
+    if (reviews === 0) return 0
+    const total = reviews.reduce((acc, curr) => {
+      return acc + curr['rating']
+    }, 0)
+    return total / reviews.length
+  }
+
+  // returns 5 star components with an active class depending on the rating
+  mapStars () {
```
Comment (eliasmalik) [inline frontend/src/js/Ratings/FiveStars.jsx:6]: Potential 2-liner:

``` javascript
const avg = Math.ceil(this.getAverageRating())
return [1, 2, 3, 4, 5].map(i => (i > avg) ? <Star active={true} /> : <Star active={false} />)
```

But if you feel this is less clear, then ignore it.

File: README.md [hunk 2]
```diff
@@ -4,17 +4,22 @@
...
 
 
 # Technology used
 
+* React.js
+* Hapi.js
+* Node.js
+* Redis
 
+(the RHNR stack? :joy:)
```
Comment (eliasmalik) [inline README.md:22]: RRHiNo stack? **R**eact **R**edis **H**ap**i** **No**de. I think we should open an issue about finding the best acronym xD

File: frontend/src/js/Ratings/FiveStars.jsx (hunk 1 above)
Comment (andrewMacmurray) [inline frontend/src/js/Ratings/FiveStars.jsx:6]: No that's much clearer, will add it

File: README.md (hunk 2 above)
Comment (andrewMacmurray) [inline README.md:22]: Brilliant, the RHINO stack rhino

**Comment by eliasmalik** [comment]
@andrewMacmurray update the branch?

File: frontend/src/js/ReviewBox/ReviewBox.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    this.state = {
+      review: '',
+      author: '',
+      rating: 0
+    }
+  }
+
+  handleAuthor (e) {
+    e.preventDefault()
+    this.setState({
```
Comment (eliasmalik) [inline frontend/src/js/ReviewBox/ReviewBox.jsx:20]: Does this work without binding in the constructor? How come?

Comment (andrewMacmurray) [inline frontend/src/js/ReviewBox/ReviewBox.jsx:20]: No reason, @eleanorreem likes binding them lower down :smile:

**Pull request merged**